package.json

```diff
@@ -47,14 +51,19 @@
     "preplaywright": "rm -rf playwright-coverage/ .nyc_output/",
     "playwright": "NODE_ENV=playwright nyc playwright test --reporter=list",
     "playwright:ui": "npx playwright test --ui",
-    "postplaywright": "npx nyc report --reporter=json --reporter=lcov --reporter=clover --report-dir=playwright-coverage"
+    "postplaywright": "npx nyc report --reporter=json --reporter=lcov --reporter=clover --report-dir=playwright-coverage",
+    "deploy-infrastructure": "cd cdk/earthdata-search-infrastructure && npm ci --include=dev && npm run cdk deploy -- --progress events --require-approval never",
+    "deploy-application": "cd cdk/earthdata-search && npm ci --include=dev && npm run cdk deploy -- --progress events --require-approval never",
+    "deploy-static": "cd cdk/earthdata-search-static && npm ci --include=dev && npm run cdk deploy -- --progress events --require-approval never"
   },
   "devDependencies": {
```

Comment:
```
npm install --include=optional sharp
npm i @rollup/rollup-darwin-x64
```
I needed to install these two locally to get my env to work. The rollup I think was a originally coming in from serverless framework